packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -211,13 +248,30 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
       onMouseMove={(e) => onHandleDragMove(e)}
       onMouseUp={() => onHandleDragEnd()}
     >
-      <div className='iot-resizable-panes-pane iot-resizable-panes-pane-left' style={{ width: `${leftPaneWidth}px` }}>
-        {leftPane}
+      <div
+        className='iot-resizable-panes-pane iot-resizable-panes-pane-left'
+        style={{
+          width: LEFT_PANE_COLLAPSED ? `${DEFAULT_COLLAPSED_SIDE_PANE_WIDTH}px` : `${leftPaneWidth}px`,
+        }}
+        onClick={() => onLeftCollapsedPaneClick()}
+      >
+        {LEFT_PANE_COLLAPSED ? (
+          <CollapsiblePanel icon={resourceExplorerPanelIcon} dataCy='collapsed-left-panel-icon' />
+        ) : (
+          leftPane
+        )}
       </div>
 
       <div
-        className='iot-resizable-panes-handle iot-resizable-panes-handle-left'
-        style={{ width: `${HANDLE_WIDTH}px` }}
+        className={
+          LEFT_PANE_COLLAPSED
+            ? 'iot-resizable-panes-handle-left'
+            : 'iot-resizable-panes-handle iot-resizable-panes-handle-left'
+        }
+        style={{
+          width: `${HANDLE_WIDTH}px`,
```

Comment:
can this Handle_width be a cloudscape token instead of a constant ? try with 8px and 12px, update with which makes it look similar to 10px

Reply:
removed this style, not required, width is taking in gridTemplateColumns